tests/test_makeps3iso.py

```
@@ -776,6 +964,146 @@ async def test_process_job_rejects_existing_split_set_without_overwrite(tmp_path
+        "chd_volumes",
+        str(tmp_path / "volume"),
+    )
+    monkeypatch.setattr(
+        job_manager_module.is_safe_directory_tree.__globals__["settings"],
+        "data_mount_root",
+        str(tmp_path / "volume"),
+    )
+
+    called = False
+
+    async def fake_convert(*_args, **_kwargs):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `fake_convert`

(ANN202)